lib/ingres_sa_dialect/base.py

```diff
@@ -686,12 +688,13 @@ def has_sequence(self, connection, sequence_name, schema=None):
                 rs.close()
     
     def get_default_schema_name(self, connection):
-        sqltext = """SELECT dbmsinfo('username')"""
-        
         rs = None
         try:
-            # TODO consider using exec_driver_sql() instead
-            rs = connection.execute(sqltext)
+            if (int((sqlalchemy.__version__).split('.')[0]) >= 2):
```

Comment:
great! consider:

    tuple(map(int, __version__.split('.'))) > (0, 1)

I can't recall off the top of my head if this works with lists. Definitely works with tuples

Reply:
Changed code to use tuple/map as suggested.